contracts/mock/MockMinter.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import "./../utils/GlobalRateLimitedMinter.sol";
+
+contract MockMinter {
+    GlobalRateLimitedMinter globalRateLimitedMinter;
```

Comment:
Is it worthwhile making this state variable visibility explicit? It's `internal` by default sure - what convention for explicit/implicit do we use normally?

Reply:
It's a mock contract so I don't think it's worth it